parameter_files/fates_params_default.cdl

```diff
@@ -529,9 +529,12 @@ variables:
 	double fates_canopy_closure_thresh ;
 		fates_canopy_closure_thresh:units = "unitless" ;
 		fates_canopy_closure_thresh:long_name = "tree canopy coverage at which crown area allometry changes from savanna to forest value" ;
+	double fates_cohort_age_tracking ;
+		fates_cohort_age_tracking:units = "0 or 1" ;
```

Comment:
I kind-of feel like this would work better, for the sake of constancy, as a 'use' flag with .true. and .false. (e.g. as for SPITFIRE and HYDRO), but I appreciate that that's probably a bigger question (as per the discussion elsewhere in the review).

Reply:
I agree. The rule sets for when size or age mortality or cohort age tracking were on or off was confusing. I've removed the cohort_age_tracking parameter (hopefully before it goes in the next set of parameter file updates) and have made cohort age tracking a flag like spitfire or hydro.

Reply:
cool...